src/code/z_lifemeter.c

```diff
@@ -172,18 +249,24 @@ s32 func_80078E84(GlobalContext* globalCtx) {
     } else {
         interfaceCtx->unk_228 = 0x140;
         interfaceCtx->unk_226 -= 0x10;
-
         if (interfaceCtx->unk_226 <= 0) {
             interfaceCtx->unk_226 = 0;
             globalCtx->damagePlayer(globalCtx, -(gSaveContext.health + 1));
             return 1;
         }
     }
-
     return 0;
 }
 
 void Health_Draw(GlobalContext* globalCtx) {
+    static u8* sHeartTextureList[] = {
```

Comment:
For these arrays, I suggest Textures (plural) instead of TextureList, for consistency with the rest of the codebase.
And imo they would be better left outside the function, especially since they will take a lot more space once they have proper symbols with names.
```suggestion
    static u8* sHeartTextures[] = {
```